okio/src/unixMain/kotlin/okio/-UnixPosixVariant.kt

```diff
@@ -108,7 +108,18 @@ internal actual fun PosixFileSystem.variantOpenReadOnly(file: Path): FileHandle
 }
 
 @ExperimentalFileSystem
-internal actual fun PosixFileSystem.variantOpenReadWrite(file: Path): FileHandle {
+internal actual fun PosixFileSystem.variantOpenReadWrite(
+  file: Path,
+  mustCreate: Boolean,
+  mustExist: Boolean
+): FileHandle {
+  require(!mustCreate || !mustExist) {
+    "Cannot require mustCreate and mustExist at the same time."
+  }
+  // We are non-atomically doing it because [fopen] doesn't any `mode` supporting either.
```

Comment:
Hmmm. . . . I was expecting we could use different flags, like you’re doing in Node on Unix?

Reply:
I cannot find anything doing either for `fopen`
https://man7.org/linux/man-pages/man3/fopen.3.html